README.md

```diff
@@ -0,0 +1,61 @@
+# Microformats JavaScript/Node Parser Website
+
+Website for Microformats JavaScript parser (based on pin13.net by [@aaronpk](https://github.com/aaronpk)).
+
+https://node.microformats.io
+
+## Deployment
+
+All commits to the `main` branch get auto-deployed to the live website [running on Heroku](https://node.microformats.io).
+
+## Getting Started
+
+This website is built using [Yarn](https://yarnpkg.com/).
+
+To start working with this project, first clone the repository for this project:
+
+```
+git clone https://github.com/microformats/microformats-parser-website-node.git
+cd microformats-parser-website-node
+```
+
+Next, install the required dependencies and start the server:
+
+```
+yarn install
+yarn run start
+# Or, if you'd like a different port:
+# PORT=5000 yarn run start 
+```
+
+You can view your running local application at this URL:
+
+```
+http://localhost:9080
+```
+
+## Requirements
+
+- [Node 17](https://nodejs.org/en/blog/release/v17.0.0/)
+- [Yarn](https://yarnpkg.com/cli/install)
+
+## Contributions
+
+1. Fork it
+2. Get it running (see Installation above)
+3. Create your feature branch (`git checkout -b my-new-feature`)
+4. Commit your changes (`git commit -am 'Add some feature'`)
+5. Push to the branch (`git push origin my-new-feature`)
+6. Create new Pull Request
+
+If you find bugs, have feature requests or questions, please
+[file an issue](https://github.com/microformats/microformats-parser-website-node/issues).
+
+## License
+
+Microformats Parser Website Rust is dedicated to the public domain using Creative Commons -- CC0 1.0 Universal.
+
+http://creativecommons.org/publicdomain/zero/1.0
+
+## Contributors
+- [Jacky Alciné](https://github.com/jalcine) &mdash; https://jacky.wtf/about
```

Comment:
While I like thanking contributors, I feel like it's hard to keep updated manually. Github's own contributor UI is probably a better tool in this case.

Reply:
I agree here, and I'll remove this section. It's going to be painful to maintain and will never reflect the true current state of things.